### PR TITLE
Add a random sentinel to close channel when terminates

### PR DESCRIPTION
**Pull request by cdelafuente-r7**
Adding a random sentinel value in the Powershell script to terminate the channel when it is done. This will prevent the execution to hang. Note that it is just a workaround, the Powershell process will still be be running on the remote target.

### CODE REVIEW

File: modules/exploits/windows/local/ms16_032_secondary_logon_handle_privesc.rb [hunk 1]
```diff
@@ -160,6 +162,7 @@ def exploit
 
       while(d = r.channel.read)
         print(d)
+        break if d =~ /#{end_flag}/
```
Comment (acammack-r7) [inline modules/exploits/windows/local/ms16_032_secondary_logon_handle_privesc.rb]:
```suggestion
        break if d.include? end_flag
```

File: modules/exploits/windows/local/ms16_032_secondary_logon_handle_privesc.rb [hunk 2]
```diff
@@ -126,6 +126,8 @@ def exploit
     ms16_032.gsub!("$cmd","\"#{cmdstr}\"")
     #lpcommandLine - capped at 1024b
     ms16_032.gsub!("$args1","\"#{psh_cmd}\"")
+    end_flag = Rex::Text.rand_text_alphanumeric(32)
+    ms16_032.gsub!("$end","\"#{end_flag}\"")
```
Comment (acammack-r7) [inline modules/exploits/windows/local/ms16_032_secondary_logon_handle_privesc.rb]: Does this need an extra set of escaped quotes with `$end` quoted in the powershell script?

File: modules/exploits/windows/local/ms16_032_secondary_logon_handle_privesc.rb (hunk 1 above)
Comment (cdelafuente-r7) [reply modules/exploits/windows/local/ms16_032_secondary_logon_handle_privesc.rb]: Yes, it is better. Thanks!

File: modules/exploits/windows/local/ms16_032_secondary_logon_handle_privesc.rb (hunk 2 above)
Comment (cdelafuente-r7) [reply modules/exploits/windows/local/ms16_032_secondary_logon_handle_privesc.rb]: You're right, they're not needed. Thanks!

**Pull request merged**